src/screens/Itinerary/TripItinerary.tsx

```diff
@@ -43,6 +44,29 @@ export default function TripItinerary(props: Props): JSX.Element {
         props.navigation.navigate(Routes.ADD_EDIT_ACTIVITY_GROUP, routeOpts);
         console.log("Add activity group button pressed");
     };
+
+    props.navigation.setOptions({
+        headerBackTitleVisible: false,
+        // eslint-disable-next-line react/display-name
+        headerTitle: (props2) => (
```

Comment:
I think props2 is a bit bad naming, headerProps or so would maybe be better I think
```suggestion
        headerTitle: (headerProps) => (
```